services/QuillLMS/client/app/bundles/Staff/components/comprehension/activitySessions/sessionView.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from "react";
+import { Link, NavLink, Redirect, Route, Switch, withRouter } from 'react-router-dom';
+import { useQuery } from 'react-query';
+
+import PromptTable from './promptTable';
+import SessionOverview from './sessionOverview';
+
+import { Error, Spinner } from '../../../../Shared/index';
+import { fetchActivity, fetchActivitySession } from '../../../utils/comprehension/activityAPIs';
+import { getPromptForActivitySession } from "../../../helpers/comprehension";
+import { BECAUSE, BUT, SO } from '../../../../../constants/comprehension';
+
+const SessionView = ({ match }) => {
+  const { params } = match;
+  const { activityId, sessionId } = params;
+
+  // cache activity data for updates
+  const { data: activityData } = useQuery({
+    queryKey: [`activity-${activityId}`, activityId],
+    queryFn: fetchActivity
+  });
+
+  // cache session data for updates
+  const { data: sessionData } = useQuery({
+    queryKey: [`activity-${activityId}-session-${sessionId}`, sessionId],
+    queryFn: fetchActivitySession
+  });
+
+  if(!sessionData) {
+    return(
+      <div className="loading-spinner-container">
+        <Spinner />
+      </div>
+    );
+  }
+
+  if(sessionData && sessionData.error) {
+    return(
+      <div className="error-container">
+        <Error error={`${sessionData.error}`} />
```

Comment:
probably can just do `error={sessionData.error}` since there's nothing else in the string.